tests/scancode/test_cli.py

```diff
@@ -463,3 +464,17 @@ def test_scan_can_run_from_other_directory():
         print(stderr)
     assert rc == 0
     check_scan(test_env.get_test_loc(expected_file), result_file, strip_dates=True, regen=False)
+
+
+class TestCLI(TestCase):
+
+	def test_shorten_filename(self):
+	    assert '0123456789...0123456789.c' == cli.shorten_filename('0123456789012345678901234567890123456789.c')
+	    assert '012345.c' == cli.shorten_filename('012345.c')
+	    assert '012345678901234567890.cpp' == cli.shorten_filename('012345678901234567890.cpp')
```

Comment:
@pombredanne what should be the output of shorten filename if a list is passed as the filename?